app/models/patient.rb

```diff
@@ -156,7 +159,10 @@ class Patient < ApplicationRecord
   before_update :set_time_zone, if: proc { |patient|
     patient.monitored_address_state_changed? || patient.address_state_changed?
   }
-  before_create :set_time_zone
+  before_create do
+    set_time_zone
+    self.enrolled_isolation = self.isolation
```

Comment:
```suggestion
    self.enrolled_isolation = isolation
```
We must sacrifice our `self` to please rubocop

Reply:
good catch: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/0861256945ec121433441c0d1342f1891e8ef915